src/app/screens/nftDashboard/receiveNft/index.tsx

```diff
@@ -133,15 +137,15 @@ function ReceiveNftModal({ visible, onClose, isGalleryOpen, setOrdinalReceiveAle
   );
 
   return isGalleryOpen ? (
-    <>
+    <VerifyButtonsContainer>
       <RowContainer>
         <Text>{t('RECEIVE_NFT')}</Text>
         <ButtonImage onClick={onReceiveModalClose}>
           <img src={Cross} alt="cross" />
         </ButtonImage>
       </RowContainer>
       {isReceivingAddressesVisible ? receiveContent : verifyOrViewAddresses}
-    </>
+    </VerifyButtonsContainer>
```

Comment:
VerifyButtonsContainer name is misleading, as it sometimes contains receiveContent and no verify buttons

why not just set a minWidth on VerifyButtonContainer, which existed already? then you don't need an extra div